package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-map-link",
-  "version": "2.11.0",
+  "version": "2.11.1",
```

Comment:
Mind removing this from the PR? Updating the version number will be done during my release flow.

Reply:
Fixed